clients/cloud/clojure/src/io/confluent/examples/clients/clj/consumer.clj

```diff
@@ -11,28 +11,28 @@
 (defn- build-properties [config-fname]
   (with-open [config (jio/reader config-fname)]
     (doto (Properties.)
-      (.putAll {ConsumerConfig/GROUP_ID_CONFIG, "clojure_example_group"
-                ConsumerConfig/KEY_DESERIALIZER_CLASS_CONFIG "org.apache.kafka.common.serialization.StringDeserializer"
+      (.putAll {ConsumerConfig/GROUP_ID_CONFIG,                "clojure_example_group"
+                ConsumerConfig/KEY_DESERIALIZER_CLASS_CONFIG   "org.apache.kafka.common.serialization.StringDeserializer"
                 ConsumerConfig/VALUE_DESERIALIZER_CLASS_CONFIG "org.apache.kafka.common.serialization.StringDeserializer"})
       (.load config))))
 
 (defn consumer! [config-fname topic]
   (with-open [consumer (KafkaConsumer. (build-properties config-fname))]
     (.subscribe consumer [topic])
-    (loop [tc 0
+    (loop [tc      0
```

Comment:
nit: afaik, unlike some languages Clojure doesn't have an official coding style. Most Clojure code I have previously seen, including everything in the `clojure` github org only has a single space between the binding form and bound expression in the bindings vector, so aligning the init-exprs here looks a bit strange to me. But this is a personal perference.